app.js

```diff
@@ -11,17 +14,18 @@ app.get("/api/topics", getTopics);
 
 app.get("/api/articles/:article_id", getArticle);
 
+app.get("/api/articles", getAllArticles);
+
 app.use((err, req, res, next) => {
   if (err.msg) {
-    res.status(404).send({ msg: "Not Found" });
+    res.status(404).send({ msg: "Article id is not found" });
   }
   next(err);
 });
 
 app.use((err, req, res, next) => {
-  console.log(err);
   if (err.code) {
-    res.status(400).send({ msg: "Bad request" });
+    res.status(400).send({ msg: "Article id does not exist" });
```

Comment:
this looks like for dealing with invalid article id, the message can be invalid input - it may not only be dealing with invalid article id, but also comment id etc.